nodebuilder/tests/swamp/swamp.go

```diff
@@ -295,7 +300,7 @@ func (s *Swamp) NewNodeWithStore(
 
 	switch tp {
 	case node.Bridge:
-		host, port, err := net.SplitHostPort(s.ClientContext.GRPCClient.Target())
+		host, port, err := net.SplitHostPort(s.ClientContext.GRPCClient.Target()) // TODO(chatton) should this be comet address or app address?
```

Comment:
I'm going to assume its comet/core. 
what is it configured to right now? if tests are passing maybe its configured correctly?

Reply:
right now it is configured to the app gRPC endpoint, since tests are passing I also assume it is fine, but I want this to show up in the final diff to ensure that is correct.